app/models/visitors_step.rb

```diff
@@ -79,9 +84,41 @@ def validate_email
     end
   end
 
+  # rubocop:disable Metrics/AbcSize
+  # rubocop:disable Metrics/MethodLength
   def validate_visitors
-    ages = visitors.map(&:age).compact
-    visitor_constraints.validate_visitor_ages_on self, :general, ages
-    visitor_constraints.validate_visitor_number self, :general, visitors.size
+    # It's invalid if there are no visitors, but there's no need to call the API
+    if visitors.empty?
+      errors.add :general, :too_few_visitors
+      return
+    end
+
+    result = PrisonVisits::Api.instance.validate_visitors(
+      lead_date_of_birth: lead_visitor.date_of_birth,
+      dates_of_birth: visitors.map(&:date_of_birth)
```

Comment:
We'll need the prison also in the API, otherwise pvb2 can't know what adult age to use